### PR TITLE
a84cc36 (linok9757, 31 minutes ago)

### PR DESCRIPTION
**Pull request by linok9757**
yoDownet: create recipe

### CODE REVIEW

**Comment by Begasus** [comment]
Still some tab's on empty lines, use tabs for indentations instead of spaces

File: net-misc/yodownet/yodownet-2014.03.09.recipe [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Alir3z4/yoDownet/archive/$portVersion.tar.gz"
+SOURCE_DIR="yoDownet-$portVersion"
+CHECKSUM_SHA256="ebe872bdb0a076968bf2d5078966513e035e05c80c140cc1e2cdf980bbf87b2f"
+PATCHES="yodownet-$portVersion.patchset"
+ADDITIONAL_FILES="
+	yodownet.rdef.in"
+
+ARCHITECTURES="x86 !x86_gcc2 x86_64"
```
Comment (Begasus) [inline net-misc/yodownet/yodownet-2014.03.09.recipe]: Suggestion:
```diff
-ARCHITECTURES="x86 !x86_gcc2 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

**Comment by linok9757** [comment]
Why you mark "x86" as "?x86" ?
--
Отправлено из Mail.Ru для Android суббота, 09 марта 2019г., 09:34 +03:00 от Schrijvers Luc  notifications@github.com :

**Comment by Begasus** [comment]
> Why did you mark "x86" as "?x86" ?

Because we don't have an x86 architecture release from Haiku, only the 32bit (gcc2hybrid) and the 64bit (x86_64)

**Comment by Begasus** [comment]
Builds ok and runs fine on gcc2h (_x86) :+1:

**Comment by Begasus** [comment]
LGTM, if others don't have an objection on the patch it could be merged, thanks for the PR

**Pull request merged**